tools/autocorres/autocorres.ML

```diff
@@ -114,7 +114,7 @@ fun named_option parser name elem_desc=
   expect "\"=\"" elem_desc parser)
 
 (* Generic parser for "NAME = STRING ..." *)
-val named_opt = named_option (Scan.repeat Parse.text)
+val named_opt = named_option (Scan.repeat Parse.embedded)
```

Comment:
for my own knowledge: did `Parse.text` get renamed to `Parse.embedded`, or is there nuance on this one?

Reply:
Looking at the commit history, `Parse.text` was removed as part of a change which `discontinued old-style {* verbatim *} tokens`. That change then replaced uses of `Parse.text` with `Parse.embedded`, so I just did the same thing.

Reply:
Thank you!